src/main.rs

```diff
@@ -30,19 +32,35 @@ pub enum App {
 pub struct RotateOptions {
     #[clap(long = "yes", short = 'y', takes_value = false)]
     pub yes: bool,
+    /// Print output as JSON
+    #[clap(long = "json", conflicts_with = "toml")]
+    json: bool,
+    /// Print output as TOML
+    #[clap(long = "toml", conflicts_with = "json")]
+    toml: bool,
```

Comment:
I have a slight preference to it being called `format` and it having the enum of `json | toml`. Not sure what the convention other tools follow is.

Reply:
+1

Reply:
Changed the implementation so that both commands take an `Option<OutputFormat>` at `--format`